libdnf5/module/module_item.cpp

```diff
@@ -441,12 +443,13 @@ void ModuleItem::create_solvable() {
     p_impl->id = ModuleItemId(
         repo_add_solvable(pool_id2repo(pool, Id(p_impl->module_sack->p_impl->repositories[p_impl->repo_id]))));
     auto solvable = pool_id2solvable(pool, p_impl->id.id);
-    auto context = p_impl->computed_static_context.empty()
-                       ? libdnf5::utils::string::c_to_str(modulemd_module_stream_get_context(p_impl->md_stream))
-                       : p_impl->computed_static_context;
+    auto original_context = modulemd_module_stream_get_context(p_impl->md_stream);
+    auto context = p_impl->computed_static_context.empty() ? libdnf5::utils::string::c_to_str(original_context)
+                                                           : p_impl->computed_static_context;
     auto arch = modulemd_module_stream_get_arch(p_impl->md_stream);
 
-    create_solvable_worker(pool, solvable, get_name(), get_stream(), get_version_str(), std::move(context), arch);
+    create_solvable_worker(
+        pool, solvable, get_name(), get_stream(), get_version_str(), std::move(context), arch, original_context);
```

Comment:
Question - why there is `std::move(context)`?

Reply:
So that it's not needlessly copied? I'm sorry, but I don't understand why you are asking.